### ISSUE TITLE
Playlist and Request Feature Incompadible with Screen-Reading Technology

### ISSUE THREAD

**Issue by BlazinWings** [AzuraCast/AzuraCast#1394]
Let me first say that I love the idea behind this project, and I am beyond excited to use it for my own network. However, I am unable to use the playlist feature since it requires drag and drop, which is not accessible with my screen reader. Is it possible to implement an alternative interface accessible to the blind?
Thanks in advance!

**Installation method**

**Host OS (for Docker installations)**

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Relevant Logs**

**Screenshots**

**Device(s):**
 - Device:  
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

**Comment by SlvrEagle23** [comment]
@BlazinWings You mean specifically the sequential playlist reordering? That's the only function that depends on drag and drop.

**Comment by BlazinWings** [comment]
Yep, that's the one.

**Comment by SlvrEagle23** [comment]
@BlazinWings Our developer friend @Vaalyn has taken some time to build a very elegant solution to the playlist reordering page that uses buttons to accomplish the same purpose that clicking and dragging serves. He also fixed the issues with the tab ordering on the public pages for song requests.

Both changes are merged into the main codebase, so updating should fix both issues.

**Comment by BlazinWings** [comment]
Thank you so much, your efforts are greatly appreciated!
